lib/request-handler.js

```diff
@@ -29,6 +32,15 @@ const FRAGMENT_EVENTS = [
  * @param {Object} response - HTTP response stream of middleware
  */
 module.exports = function processRequest(options, request, response) {
+    const parentSpanContext = tracer.extract(
```

Comment:
tracer should be present after this.emit("start"), otherwise our overhead wont calculate the time spent in starting tracer context.

Reply:
I actually, deliberately, put it before so that the Span duration would itself contain the overhead :)
We may be having some redundant measurements for some time